<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Pavankumar" failures="0" tests="0" name="Default test" time="1.167" errors="0" timestamp="2024-01-10T11:53:00 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="@BeforeClass setUpClass" time="1.04">
    <failure type="io.cucumber.core.gherkin.FeatureParserException" message="Failed to parse resource at: file:///C:/Users/vijay/Desktop/drakshayani/testmaven/Cucucmberframework/./src/test/resources/feature/abc.feature
(5:2): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &amp;apos;Scenario Outline: login to hrm application with invalid valid test data&amp;apos;
(6:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &amp;apos;Given using hrm url in any browser&amp;apos;
(7:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &amp;apos;When user type &amp;lt;Username&amp;gt; and &amp;lt;Password&amp;gt;&amp;apos;
(8:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &amp;apos;And click on login button&amp;apos;
(9:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &amp;apos;Then verify the error message&amp;apos;
(11:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &amp;apos;Examples:&amp;apos;
(12:7): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &amp;apos;|Username      | Password |&amp;apos;
(13:7): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &amp;apos;|drakshayani   |    hasdgsajsad |&amp;apos;
(14:7): expected: #TagLine, #FeatureLine, #Comment, #Empty, got &amp;apos;| Admin        |    admin123 |&amp;apos;
(15:0): unexpected end of file, expected: #TagLine, #FeatureLine, #Comment, #Empty">
      <![CDATA[io.cucumber.core.gherkin.FeatureParserException: Failed to parse resource at: file:///C:/Users/vijay/Desktop/drakshayani/testmaven/Cucucmberframework/./src/test/resources/feature/abc.feature
(5:2): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'Scenario Outline: login to hrm application with invalid valid test data'
(6:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'Given using hrm url in any browser'
(7:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'When user type <Username> and <Password>'
(8:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'And click on login button'
(9:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'Then verify the error message'
(11:5): expected: #TagLine, #FeatureLine, #Comment, #Empty, got 'Examples:'
(12:7): expected: #TagLine, #FeatureLine, #Comment, #Empty, got '|Username      | Password |'
(13:7): expected: #TagLine, #FeatureLine, #Comment, #Empty, got '|drakshayani   |    hasdgsajsad |'
(14:7): expected: #TagLine, #FeatureLine, #Comment, #Empty, got '| Admin        |    admin123 |'
(15:0): unexpected end of file, expected: #TagLine, #FeatureLine, #Comment, #Empty
at io.cucumber.core.gherkin.messages.GherkinMessagesFeatureParser.parse(GherkinMessagesFeatureParser.java:56)
at io.cucumber.core.feature.FeatureParser.parseResource(FeatureParser.java:46)
at java.base/java.util.function.BiFunction.lambda$andThen$0(BiFunction.java:70)
at io.cucumber.core.resource.ResourceScanner.lambda$processResource$1(ResourceScanner.java:79)
at io.cucumber.core.resource.PathScanner$ResourceFileVisitor.visitFile(PathScanner.java:75)
at io.cucumber.core.resource.PathScanner$ResourceFileVisitor.visitFile(PathScanner.java:60)
at java.base/java.nio.file.Files.walkFileTree(Files.java:2786)
at io.cucumber.core.resource.PathScanner.findResourcesForPath(PathScanner.java:53)
at io.cucumber.core.resource.PathScanner.findResourcesForUri(PathScanner.java:31)
at io.cucumber.core.resource.ResourceScanner.findResourcesForUri(ResourceScanner.java:61)
at io.cucumber.core.resource.ResourceScanner.scanForResourcesUri(ResourceScanner.java:134)
at io.cucumber.core.runtime.FeaturePathFeatureSupplier.loadFeatures(FeaturePathFeatureSupplier.java:62)
at io.cucumber.core.runtime.FeaturePathFeatureSupplier.get(FeaturePathFeatureSupplier.java:45)
at io.cucumber.testng.TestNGCucumberRunner.<init>(TestNGCucumberRunner.java:122)
at io.cucumber.testng.AbstractTestNGCucumberTests.setUpClass(AbstractTestNGCucumberTests.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setUpClass -->
</testsuite> <!-- Default test -->
